server/data-loader-helpers.js

```diff
@@ -14,12 +14,12 @@ exports.catalogDataLoader = function catalogDataLoader(client, config) {
   });
 };
 
-exports.homeCollectionOrStories = function homeCollectionOrStories(client, depth = 1, getStoryLimits, params = {}) {
+exports.homeCollectionOrStories = function homeCollectionOrStories(client, depth = 1, getStoryLimits, params = {}, collectionOfCollectionsIndex = []) {
```

Comment:
Array variables should typically be plural

```suggestion
exports.homeCollectionOrStories = function homeCollectionOrStories(client, depth = 1, getStoryLimits, params = {}, collectionOfCollectionsIndexes = []) {
```